app/models/post.rb

```diff
@@ -9,4 +9,19 @@ class Post < ApplicationRecord
   has_many :comments, dependent: :destroy
   has_many :post_awards, dependent: :destroy
   has_many :awards, -> { order 'post_awards.created_at' }, through: :post_awards
+  has_many :tag_references, as: :tagable, dependent: :destroy
+
+  def tags_as_json
```

Comment:
Your time have come!

Izveido lūdzu concern un samet tur kodu priekš tags_as_json un tags= metodem
Un tad include to gan iekš Post, gan iekš Comunity